### PR TITLE
Code Quality Improvement - @Override annotation should be used on any method overriding

### PR DESCRIPTION
**Pull request by civanyp**
This pull request is focused on resolving occurrences of Sonar rule
squid:S1161 - "@Override annotation should be used on any method overriding". You can find more information about the issue here: https://dev.eclipse.org/sonar/rules/show/squid:S1161.

Please let me know if you have any questions.

Christian

### CODE REVIEW

File: pac4j-saml/src/main/java/org/pac4j/saml/transport/Pac4jHTTPPostEncoder.java [hunk 1]
```diff
@@ -92,6 +93,7 @@ protected void doDestroy() {
     }
 
     /** {@inheritDoc} */
```
Comment (mmoayyed) [inline pac4j-saml/src/main/java/org/pac4j/saml/transport/Pac4jHTTPPostEncoder.java]: You may also want to remove `/** {@inheritDoc} */` as well, for overrides.

**Comment by civanyp** [comment]
Done! Removed /** {@inheritDoc} */ for overrides change.

File: pac4j-saml/src/main/java/org/pac4j/saml/transport/Pac4jHTTPPostEncoder.java (hunk 1 above)
Comment (civanyp) [inline pac4j-saml/src/main/java/org/pac4j/saml/transport/Pac4jHTTPPostEncoder.java]: Done! Removed /** {@inheritDoc} */ for overrides change.

**Comment by mmoayyed** [comment]
Great, :+1:

File: pac4j-core/src/main/java/org/pac4j/core/http/DefaultAjaxRequestResolver.java [hunk 2]
```diff
@@ -30,6 +30,7 @@
     /**
```
Comment (leleuj) [inline pac4j-core/src/main/java/org/pac4j/core/http/DefaultAjaxRequestResolver.java]: I think the use of `@Override` allows us not to add `{@inheritDoc}`, doesn't it? The Javadoc comment can be safely removed.

Comment (civanyp) [inline pac4j-core/src/main/java/org/pac4j/core/http/DefaultAjaxRequestResolver.java]: Done! Removed all Javadoc comment {@inheritDoc} for @Override change.

**Comment by leleuj** [comment]
Thanks

**Pull request merged**